pkg/auth/authenticator/bearertoken/bearertoken.go

```diff
@@ -20,16 +20,19 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/go-openapi/spec"
```

Comment:
I'm +1 for the `GetOpenAPISecurityDefinition` logic living outside of these packages. `pkg/apiserver/authenticator` seems like a better place since that package understands the name and type (bearer, x509, etc.) of each authenticator.